steadybit/manifest.json

```
@@ -0,0 +1,70 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "b1194c36-afd0-47dc-9c0a-11f3ab82f387",
+  "app_id": "steadybit",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Immediately improve your systems' reliability with chaos engineering.",
```

```suggestion
    "description": "Immediately improve your systems' reliability with chaos engineering",
```
We usually don't include periods in short descriptions